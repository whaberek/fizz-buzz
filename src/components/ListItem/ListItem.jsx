import React from 'react';

const ListItem = ({ children }) => (
  <div>{children}</div>
);

export { ListItem };
