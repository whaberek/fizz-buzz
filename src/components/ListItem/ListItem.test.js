import React from 'react';
import { shallow } from 'enzyme';

import { ListItem } from './ListItem';

describe('ListItem', () => {
  it('should render layout with children', () => {
    const wrapper = shallow(<ListItem>foo</ListItem>);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render layout without children', () => {
    const wrapper = shallow(<ListItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
