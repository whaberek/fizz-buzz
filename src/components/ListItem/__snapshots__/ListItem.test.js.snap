// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should render layout with children 1`] = `
<div>
  foo
</div>
`;

exports[`ListItem should render layout without children 1`] = `<div />`;
